src/csvinputfileconn.cc

```diff
@@ -257,9 +257,35 @@ namespace dd
     if (!_id.empty() && !has_id)
       throw InputConnectorBadParamException("cannot find id column " + _id);
   }
+
+
+  void CSVInputFileConn::find_min_max(std::ifstream &csv_file)
```

Comment:
When calling this, we should be careful we're looking for bounds over the training *and* testing sets. This is because otherwise some values within the test set can be found to be out of bounds at test time.

Reply:
_adconf was only needed to pass "shuffle" parameter, but shuffling has been refactored a lot in order to allow to shuffle series themselves (ie order of series) but not their content (at it follows time), also to shuffle cvs (if not timeries) hence its removal here